7.1.0/oracle/org_data_fetch.sh

```
@@ -0,0 +1,79 @@
+  # Skip if either field is empty
+  [[ -z "$TENANT_ID" ]] && continue
+
+  # Write processed line to the export file
+  echo "$TENANT_ID,$ORG_UUID" >> "$EXPORT_FILE"
+done < "$RAW_OUTPUT"
+
+# Clean up temporary raw output file
+rm -f "$RAW_OUTPUT"
+
+# Check if the export file contains more than just the header
+if [[ $(wc -l < "$EXPORT_FILE") -le 1 ]]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this if expression.

(SC1009)